examples/lwt/lwt_get_pipelined.ml

```diff
@@ -33,18 +33,20 @@ let request host =
   in
   let* response = Client.get client "/" in
   let open Lwt_syntax.Async in
-  let* () =
+  let (stream, _) = Body.to_string_stream response.body in
+  let* () = 
     Lwt_stream.iter_s
```

Comment:
If you're up to it, the recommended API to traverse the stream (and know whether the body was successfully read) is `Body.iter_string_s`: https://github.com/anmonteiro/piaf/blob/master/lib/piaf.mli#L236-L239

Reply:
I used `Body.iter_string_s` in the latest commit, please have a look.